src/gso.ts

```diff
@@ -132,6 +132,7 @@ export class GSO {
   public async createConfigInstruction(
     lockupRatioPerMillionLots: number,
     optionExpiration: number,
+    subscriptionExpiration: number,
```

Comment:
subscription_period_end is the name of the variable in the program. Can we keep the naming consistent?

Reply:
complains about camel case with this variable naming

Reply:
renamed subscriptionPeriodEnd